src/plugins/implicitRelationships.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2023 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import { FluxDispatcher, ChannelStore, GuildStore, RelationshipStore, UserStore } from "@webpack/common";
+import { findStoreLazy, findByProps } from "@webpack";
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+
+const UserAffinitiesStore = findStoreLazy("UserAffinitiesStore");
+
+export default definePlugin({
+    name: "ImplicitRelationships",
+    description: "Shows your implicit relationships in the Friends tab.",
+    authors: [Devs.Dolfies],
+    patches: [
+        {
+            find: "FriendsEmptyState: Invalid empty state",
+            replacement: [
+                // Counts header
+                {
+                    match: /toString\(\)\}\);case ([^;]+)\.([^;]+)\.BLOCKED/,
+                    replace: "toString()});case $1.$2.IMPLICIT:return \"Implicit — \"+t.toString();case $1.$2.BLOCKED"
+                },
+                // No friends page
+                {
+                    match: /case ([^;]+)\.([^;]+)\.ONLINE:return ([^;]+)\.SECTION_ONLINE/,
+                    replace: "case $1.$2.ONLINE:case $1.$2.IMPLICIT:return $3.SECTION_ONLINE"
+                },
+                // Sections header
+                {
+                    match: /\(0,([^;]+)\.jsx\)\(([^;]+)\.TabBar\.Item,\{id:([^;]+)\.([^;]+)\.BLOCKED,([^\s]+)children:([^;]+)\.([^;]+)\.Messages\.BLOCKED\}\)/,
+                    replace: "(0,$1.jsx)($2.TabBar.Item,{id:$3.$4.IMPLICIT,$5children:\"Implicit\"}),$&"
+                },
+                // Sections content
+                {
+                    match: /case ([^;]+)\.([^;]+)\.BLOCKED:return ([^;]+)\.type===([^;]+)\.([^;]+)\.BLOCKED/,
+                    replace: "case $1.$2.BLOCKED:return $3.type===$4.$5.BLOCKED;case $1.$2.IMPLICIT:return $3.type===5"
+                },
+                // Piggyback relationship fetch
+                {
+                    match: /([^;]+)\.([^;]+)\.fetchRelationships\(\)/,
+                    // This relationship fetch is actually completely useless, but whatevs
+                    replace: "$1.$2.fetchRelationships(),Vencord.Plugins.plugins.ImplicitRelationships.fetchImplicitRelationships()"
```

Comment:
```suggestion
                    replace: "$1.$2.fetchRelationships(),$self.fetchImplicitRelationships()"
```

you should use `$self` instead of the full path

Reply:
Another thing I didn't know existed, thanks again!